src/pages/workspace/accounting/netsuite/export/NetSuiteInvoiceItemPreferenceSelectPage.tsx

```diff
@@ -0,0 +1,107 @@
+        value: postingPreference,
+        text: translate(`workspace.netsuite.invoiceItem.values.${postingPreference}.label`),
+        keyForList: postingPreference,
+        isSelected: config?.invoiceItemPreference === postingPreference,
+    }));
+
+    const selectInvoicePreference = useCallback(
+        (row: MenuListItem) => {
+            if (row.value !== config?.invoiceItemPreference) {
+                Connections.updateNetSuiteInvoiceItemPreference(policyID, row.value, config?.invoiceItemPreference);
+            }
+            if (row.value === CONST.NETSUITE_INVOICE_ITEM_PREFERENCE.CREATE) {
+                Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT.getRoute(policyID));
+            }
+        },
+        [config?.invoiceItemPreference, policyID],
+    );
+
+    const headerContent = useMemo(
+        () => (
+            <View style={[styles.pb2, styles.ph5]}>
+                <Text style={[styles.pb2, styles.textNormal]}>{translate(`workspace.netsuite.invoiceItem.values.${config?.invoiceItemPreference ?? 'create'}.description`)}</Text>
+            </View>
+        ),
+        [styles.pb2, styles.ph5, styles.textNormal, translate, config?.invoiceItemPreference],
+    );
+
+    return (
+        <SelectionScreen
+            displayName={NetSuiteInvoiceItemPreferenceSelectPage.displayName}
+            title="workspace.netsuite.invoiceItem.label"
+            sections={[{data}]}
+            listItem={RadioListItem}
+            headerContent={headerContent}
+            onSelectRow={(selection: SelectorType) => selectInvoicePreference(selection as MenuListItem)}
+            initiallyFocusedOptionKey={data.find((mode) => mode.isSelected)?.keyForList}
+            policyID={policyID}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT.getRoute(policyID))}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.NETSUITE}
+            listFooterContent={
+                config?.invoiceItemPreference === CONST.NETSUITE_INVOICE_ITEM_PREFERENCE.SELECT ? (
+                    <OfflineWithFeedback
+                        key={translate('workspace.netsuite.invoiceItem.label')}
+                        pendingAction={config?.pendingFields?.invoiceItem}
+                        errors={ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.INVOICE_ITEM)}
+                        errorRowStyles={[styles.ph5]}
+                        onClose={() => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.INVOICE_ITEM)}
+                    >
+                        <MenuItemWithTopDescription
+                            description={translate('workspace.netsuite.invoiceItem.label')}
+                            title={selectedItem ? selectedItem.name : undefined}
```

Comment:
If `selectedItem.name` is not defined, does it even make sense to show this `MenuItem`?

Reply:
🤦